ADBench/run-all.ps1

```diff
@@ -53,23 +53,51 @@ param(# Which build to test.
       [double]$timeout=300,    
       
       # Where to store the ouput, defaults to tmp/ in the project root 
-      [string]$tmpdir="",      
+      [string]$tmpdir="",
       
       # Repeat tests, even if output file exists
-      [switch]$repeat,         
+      [switch]$repeat,
       
       # Repeat only failed tests
       [switch]$repeat_failures,
       
       # List of tools to run
-      [string[]]$tools=@(),    
+      [string[]]$tools=@(),
+      
+      # Don't delete produced jacobians even if they're accurate
+      [switch]$keep_correct_jacobians, 
       
       # GMM D values to try.  Must be a subset of the list of
       # compiled values in ADBench/cmake-vars-$buildtype.ps1                         
       [int[]]$gmm_d_vals_param,
       
       # GMM K values to run.  As above.
-      [int[]]$gmm_k_vals_param 
+      [int[]]$gmm_k_vals_param,
+
+      # GMM sizes to try. Must be a subset of @("1k", "10k", "2.5M")
+      # 2.5M currently is not supported
+      [string[]]$gmm_sizes = @("1k", "10k"),
+
+      # Hand problem sizes to try. Must be a subset of @("small", "big")
+      [string[]]$hand_sizes = @("small", "big"),
+
+      # Number of the first BA problem to try. Must be between 1 and ba_max_n
+      [int]$ba_min_n = 1,
+
+      # Number of the last BA problem to try. Must be between ba_min_n and 20
+      [int]$ba_max_n = 5,
+
+      # Number of the first Hand problem to try. Must be between 1 and hand_max_n
+      [int]$hand_min_n = 1,
+
+      # Number of the last Hand problem to try. Must be between hand_min_n and 12
+      [int]$hand_max_n = 5,
```

Comment:
It would seem more natural to me if this was the largest possible (which seems to be 12).  Is there a good reason the default wouldn't be that?

Reply:
[See other comment for related discussion]

Reply:
See above.

Reply:
OK